### PR TITLE
Add guidelines and reference section

### PR DESCRIPTION
**Pull request by OrthoLoess**
Includes some sample content, mostly taken from dev site.

Will also contain External Resources page.

Thoughts?

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 site_name: EVE Online Third-Party Developer Documentation
 pages:
 - ['index.md', 'Home']
+- ['reference/license.md', 'Guidelines and Reference', 'Developer License']
```
Comment (Regner) [inline mkdocs.yml]: Should these be at the top? Seems like something that should be at the bottom.

Comment (OrthoLoess) [inline mkdocs.yml]: The only thing is that the bottom is a long way down...

As in, you have to scroll to see it with the window size I use.

File: mkdocs.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+- ['reference/guidelines.md', 'Guidelines and Reference', 'Guidelines']
```
Comment (Regner) [inline mkdocs.yml]: These 3 entries should probably say References plural.

File: mkdocs.yml (hunk 1 above)
Comment (Regner) [inline mkdocs.yml]: Yea, thats an issue we are going to have to deal with in general though. Personally I would intuitively think to look at the end to find this kind of information myself. However, for now I will accept it like this and we can continue thinking about it going forward.

File: mkdocs.yml (hunk 2 above)
Comment (OrthoLoess) [inline mkdocs.yml]: I disagree, but not strongly enough to argue, so changed :)

**Pull request merged**